source/index.html.md

```diff
@@ -156,53 +174,67 @@ On the right you can see how to generate a `keypair` with the `Stellar JS SDK` a
   ]
 }
 ```
-The Stellar network allows us to represent any kind of asset. All assets
-in Stellar can be traded and exchanged with each other.
 
-Like other protocols, Stellar has a native asset which is called the
-`Lumen` represented with the symbol `XLM`. Stellar accounts can hold
-multiple assets as long as they trust the asset and in some
-cases they have been authorized to hold the asset.
+[SUGGESTION: The paragraph feels a bit disorganized. I'd start with a definition and
+then move to how Stellar defines its assets, let me know what you think:]
 
-Any account can create their own asset representing traditional or custom (work/usage/hybrid) of assets.
+Assets are the things you normally associate with resources. Things that have value,
+like fiat, equity, real estate, goats, you name it. Cryptocurrencies allows us to
+represent any kind of asset. Stellar is no exception, it allows any account to define
+traditional assets or even come up with new exciting assets. These can be easily
+traded and exchanged over the Stellar network.
 
-Traditional assets are a cryptographic representation of things like
-fiat, equity, real estate, goats, you name it.
+[NOTE: What is fiat? People not familiar with Crypto/finances might have a hard time]
 
-Lumens are an example of a custom asset or token, at they allow us to
-interact with the Stellar network. There are many other types of
-assets built on top of Stellar, one example is the `EURT` which is a
-representation of the `EURO` and allows people to do cross-border
-remittances without incurring in high transaction fees. There is also
-`MOBI` which allows people to use the `Mobius network`, if you hold `MOBI` then
-your can interact with the applications in their network.
+Under the blankets, Stellar uses a native asset which is called the `Lumen`
+represented with the symbol `XLM`. Stellar accounts can hold multiple assets as long
+as they "trust" the asset and in some cases they have been authorized to hold the
+asset.
 
-On the right you can see the JSON representation of a Stellar
-account. Each account has a key called balances, representing the
-assets held by the account.
+[NOTE: What does it mean to "Trust" the asset?]
```

Comment:
good point, I think at this point we can remove that and say  that you can hold multiple assets.